### PR TITLE
Dealing with line endings

### PR DESCRIPTION
**Pull request by robocoder**
see https://help.github.com/articles/dealing-with-line-endings/

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+*.MF text
+*.mf text
+*.commons-logging text
+*.config text
+*.plist text
+*.properties text
+*.yml text
+*.schema text
+*.script text
+*.weeklytext
```
Comment (thc202) [inline .gitattributes]: There's a missing space.

Comment (robocoder) [inline .gitattributes]: Thanks.

**Comment by kingthorin** [comment]
Is this still relevant?

**Comment by robocoder** [comment]
It's an individual preference whether or not the codebase has mixed line endings.

On the `develop` branch, adding .gitattributes, I see (via `git diff`) 200+ files where CRLF line endings are converted to LF.

**Comment by thc202** [comment]
@robocoder could you enable the option "Allow edits from maintainers" ? [1]

[1] https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/

**Comment by robocoder** [comment]
Ok

**Comment by thc202** [comment]
Thank you!

**Comment by thc202** [comment]
Updated the PR:
 - Rebased to include latest changes;
- Tweaked the gitatributes to always use LF except for specific files (to make reproducible builds easier), keep the files in alphabetic order, and add more files (text and binary);
- Normalise line endings in the files.

**Pull request merged**